app/controllers/transactions_controller.rb

```diff
@@ -21,7 +23,30 @@ def edit
 
   # POST /transactions
   def create
-    @transaction = Transaction.new(transaction_params)
+    food = Food.find(params[:transaction][:food_id])
+    p food.inspect
+    seller = food.seller
+    buyer = current_user
+    @transaction = Transaction.new(food: food, seller: seller, buyer: buyer)
+
+    package = {
+      manifest: "a box of kittens",
+      pickup_name: "The Warehouse",
+      pickup_address: food.seller_location,
+      pickup_phone_number: "555-555-5555",
+      pickup_business_name: "Optional Pickup Business Name, Inc.",
+      pickup_notes: "Optional note that this is Invoice #123",
+      dropoff_name: "Alice",
+      dropoff_address: params[:transaction][:buyer_location],
+      dropoff_phone_number: "415-555-1234",
+      dropoff_business_name: "Optional Dropoff Business Name, Inc.",
+      dropoff_notes: "Optional note to ring the bell"
+      #quote_id: "dqt_K9LFfpSZCdAJsk" # optional
```

Comment:
Incorrect indentation detected (column 6 instead of 8).<br>Missing space after #.